setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import find_packages
 from setuptools import setup
 
-from triangular_arbitrage import PROJECT_NAME, VERSION
+from arbitrage_opportunity import PROJECT_NAME, VERSION
```

Comment:
Although the library now offers more than just triangular arbitrage, could you please keep the old package name? 

This will break the imports for people (including us) who already use the library.

Reply:
I see. Will revert the renames for now. Maybe something in the future if big updates/overhaul are required to avoid confusions.

Reply:
I totally agree on that.